### DISCUSSION TITLE
Server names bolded on directory servers

### DISCUSSION THREAD

**Discussion by rdica** [jamulussoftware/jamulus#2070] [General]
I noticed some servers names in directory service listings are not bolded.

What determines when/why a server name is bolded?

Thanks

**Comment by gilgongo** [comment]
It's if they've been online for more than 24 hours I think.

**Comment by pcar75** [comment]
Form SourceForge discussion ...
" Volker Fischer - 2020-04-15
If a server is longer registered than 24 hours, it will shown in bold font. "

**Comment by rdica** [comment]
As of this writing, it's been 25 hrs since server was registered on Any Genre 2, and it's not in bold, guess that's not it...

**Reply by gilgongo** [reply]
I guess at this point we may need to examine the source code :-) Is says "longer than" 24 hours here: https://jamulus.io/wiki/Software-Manual#connectdisconnect-button

**Comment by rdica** [comment]
Found this in `global.h`:

    // defines the minimum time a server must run to be a permanent server
    #define SERVLIST_TIME_PERMSERV_MINUTES 2880 // minutes, 2880 = 60 min * 24 h * 2 d

so 48h, but does this also apply to the directory servers currently in use for the various genres?

**Reply by gilgongo** [reply]
Interesting - better update the documentation then! I suppose it also applies to the directories.

**Comment by rdica** [comment]
Confirmed with my server, after 48 hrs its now showing in bold.
